### ISSUE TITLE
wrong loss value logged in the pytorch 1.1 version

### ISSUE THREAD

**Issue by lijing1996** [HRNet/HRNet-Semantic-Segmentation#34]
As in lib/core/function.py
https://github.com/HRNet/HRNet-Semantic-Segmentation/blob/06142dc1c7026e256a7561c3e875b06622b5670f/lib/core/function.py#L35
you use the default 'sum' op to reduce the tensors across different processes,
>         reduced_loss = reduce_tensor(loss)

>         ave_loss.update(reduced_loss.item())
here, the loss value logged is the sum over different processes, but it should be averaged
